### PR TITLE
[SPARK-27444][SQL][FOLLOWUP][MINOR][TEST] Add a test for describing multi select query.

### PR DESCRIPTION
**Pull request by dilipbiswal**
## What changes were proposed in this pull request?
This is a minor pr to add a test to describe a multi select query.

## How was this patch tested?
Added a test in describe-query.sql

### CODE REVIEW

**Comment by dilipbiswal** [comment]
cc @cloud-fan

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala [hunk 1]
```diff
@@ -635,7 +635,9 @@ case class DescribeTableCommand(
  * 3. VALUES statement.
  * 4. TABLE statement. Example : TABLE table_name
  * 5. statements of the form 'FROM table SELECT *'
- * 6. Common table expressions (CTEs)
+ * 6. Multi select statements of the following form:
```
Comment (beliefer) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala]: I think 'nested' is better than 'multi'.

Comment (cloud-fan) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala]: `multi` refers to `select * select *`, nested is not the key here.

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Comment by dilipbiswal** [comment]
@cloud-fan Thank you very much.

**Pull request closed without merging**